### PR TITLE
Adding a flag to have the option of chosing runtime for k8s cluster

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This addition is part of the story https://github.ibm.com/powercloud/container-dev/issues/1353

Need "runtime" flag in kubetest2 to pass the choice of runtime(docker/containerd as of now) to the k8s-ansible sub-module it uses to build the test cluster.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/26#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/providers/common/provider.go
Comment (mkumatag) [inline pkg/providers/common/provider.go:36]: Let's set the `docker` as a default value as of now, will change to containerd or something in future..

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: pkg/providers/common/provider.go
Comment (Rajalakshmi-Girish) [reply pkg/providers/common/provider.go:36]: The variable in k8s-ansible has the default value as docker.
As we dont have default values here for variables s3_server, bucket, directory, I thought we needn't have for runtime as well.

Comment (Rajalakshmi-Girish) [reply pkg/providers/common/provider.go:36]: @mkumatag ^^

Comment (mkumatag) [reply pkg/providers/common/provider.go:36]: make sense

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/26#issuecomment-739814116" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/26#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
